### DISCUSSION TITLE
PDF output

### DISCUSSION THREAD

**Discussion by ncamachoh** [thegetty/quire#712] [Get Help]
Hello!
I am trying to use the pdf output option that was recently enabled, but I get a "pdf.html" instead of .pdf.
I don't know if should do something differently or update Quire to the latest version to get the pdf output functionality. Has anybody created a pdf from a v1 project?

Thanks!

PS C:\Users\nataliac\Documents\Github\vag-heath-carr> quire build
[CLI] running eleventy using lib/11ty cli
[CLI:11ty] running eleventy build
[CLI:11ty] projectRoot C:\Users\nataliac\Documents\Github\vag-heath-carr
[CLI:11ty] {
  config: 'C:\\Users\\nataliac\\Documents\\Github\\vag-heath-carr\\.eleventy.js',
  input: 'content',
  output: '_site',
  epub: '_epub',
  data: '_computed',
  includes: '..\\_includes',
  layouts: '..\\_layouts',
  public: './public'
}
[quire] DEBUG    Figures:ImageProcessor
      inputRoot: C:\Users\nataliac\Documents\Github\vag-heath-carr\content\_assets\images
      outputRoot: C:\Users\nataliac\Documents\Github\vag-heath-carr\public
 
[quire] INFO     Figures                        Processing complete
→ _site/index.html                     ./content/index.md                                       29.3kB --
→ _site/bibliography/index.html        ./content/bibliography.md                                32.2kB --
→ _site/biographies/index.html         ./content/biographies.md                                 31.7kB --
→ _site/colophon/index.html            ./content/colophon.md                                    31.4kB --
→ _site/contents/index.html            ./content/contents.md                                    59.6kB --
→ _site/contributors/index.html        ./content/contributors.md                                34.1kB --
→ _site/edith-heath/index.html         ./content/edith-heat[…]/index.md                         43.3kB --
→ _site/edith-heath-in[…]/index.html   ./content/edith-heat[…]/408-timeline.md                  98.9kB --
→ _site/edith-heath/20[…]/index.html   ./content/edith-heat[…]/203-coupe-original-glazes.md     49.5kB --
→ _site/emily-carr/index.html          ./content/emily-carr/index.md                            37.7kB --
→ _site/emily-carr/301[…]/index.html   ./content/emily-carr/301-oil-on-paper.md                 68.3kB --
→ _site/emily-carr/302[…]/index.html   ./content/emily-carr/302-charcoal.md                     49.6kB --
→ _site/emily-carr/303[…]/index.html   ./content/emily-carr/303-forest-paintings.md             50.0kB --
→ _site/foreword/index.html            ./content/foreword.md                                    35.5kB --
→ _site/introduction/index.html        ./content/introduction.md                                35.4kB --
→ _site/pdf-epub-copyr[…]/index.html   ./content/pdf-epub-copyright.md                          25.5kB --
→ _site/pdf-epub-half-[…]/index.html   ./content/pdf-epub-half-title.md                         24.2kB --
→ _site/pdf-epub-title/index.html      ./content/pdf-epub-title.md                              24.4kB --
[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] EPERM: operation not permitted, rename 'C:\Users\nataliac\Documents\Github\vag-heath-carr\_site' -> 'C:\Users\nataliac\Documents\Github\vag-heath-carr\.11ty-vite' (via Error)
[11ty]
[11ty] Original error stack trace: Error: EPERM: operation not permitted, rename 'C:\Users\nataliac\Documents\Github\vag-heath-carr\_site' -> 'C:\Users\nataliac\Documents\Github\vag-heath-carr\.11ty-vite'
[11ty] Benchmark   3736ms  13%    61× (Configuration) "slugifyIds" JavaScript Function
[11ty] Benchmark   3680ms  12%    30× (Configuration) "epub" Transform
[11ty] Benchmark   2954ms  10%    30× (Configuration) "pdf" Transform
[11ty] Benchmark   5121ms  17%    30× (Configuration) "html" Transform
[11ty] Copied 247 files / Wrote 30 files in 26.39 seconds (879.7ms each, v2.0.0-canary.33)
PS C:\Users\nataliac\Documents\Github\vag-heath-carr> quire pdf
[CLI:lib/pdf] generating PDF using Paged.js
[CLI:lib/pdf/pagedjs] printing C:\Users\nataliac\Documents\Github\vag-heath-carr\_site\pdf.html

**Comment by geealbers** [comment]
@ncamachoh, sorry about that, we've had some Windows issues with 1.0.0. Are you on Windows 10 or 11? And can you check your Quire version with `quire --version` and let me know what it says?

**Reply by ncamachoh** [reply]
Hi Greg,

Yes, I'm on Windows 10 Pro and Quire version is 1.0.0-rc.0. Should I run an upgrade of Quire to generate the pdf? It won't affect the rest of the project?

**Reply by Erin-Cecele** [reply]
Hi @ncamachoh. Thank you for submitting your new Quire project! It looks great. We will feature it in this week's newsletter. As for the PDF output, there is a known issue for `quire build` in Windows 10. This was also brought to our attention by @kanotson in [discussion #684](https://github.com/thegetty/quire/discussions/684) and is related to issue #626. This has been identified as https://github.com/11ty/eleventy-plugin-vite/issues/22, a possible fix seems to have been found, and there's hopefully a forthcoming PR to address it. We will update issue #626 when we have a fix. Thank you for your patience.

**Reply by ncamachoh** [reply]
Great news, Erin! Thanks for keeping me in the loop.

**Reply by Erin-Cecele** [reply]
Hi @ncamachoh, while the issue isn't fixed yet, I'm pleased to say we have discovered a workaround: https://github.com/thegetty/quire/issues/626#issuecomment-1542935869.

**Comment by ncamachoh** [comment]
Hi Erin,

I did the workaround as explained and it worked! It generated a pdf but unfortunately images are not visible, I don't know why. I get an icon and the alt-text. Has this happened to anyone else? Or do you know what could be causing this?
Thanks!
***@***.***

**Reply by Erin-Cecele** [reply]
Thank you for letting us know @ncamachoh. Let us look into this a bit further.

**Reply by ncamachoh** [reply]
Hi Erin! I just tried this again and keep getting the same result (pdf, no images). Do you have any updates for the pdf output on Windows 10 / v. 1.0.0-rc.0. Thanks!

**Comment by Erin-Cecele** [comment]
Hi @ncamachoh We are looking into this and will get back to you with a solution a soon as possible. Thank you!
